ds3-sdk/src/main/java/com/spectralogic/ds3client/Ds3ClientBuilder.java

```diff
@@ -78,17 +78,17 @@ public static Ds3ClientBuilder create(final String endpoint, final Credentials c
      * @throws IllegalArgumentException
      */
     public static Ds3ClientBuilder fromEnv() throws IllegalArgumentException {
-        final String endpoint = System.getenv(ENDPOINT);
+        final String endpoint = System.getenv("ENDPOINT");
```

Comment:
This change is wrong since the ENDPOINT is an argument:
static final private String ENDPOINT = "DS3_ENDPOINT";

and by changing this you are doing it wrong

This comment is for all the changes you did